src/get-started/java-binding.md

```diff
@@ -0,0 +1,382 @@
+supported by both the service client and the service itself.
+
+!!! note
+    In the current framework version each service must implement a method for
+    sending transactions of this service to the network. It is planned that this
+    mechanism will be standardized and moved to the core.
+    An example of a pseudo-code transaction handler is shown below. For an
+    implementation example, see
+    `com.exonum.binding.cryptocurrency.ApiController#submitTransaction`.
+
+!!! note "Example of a Pseudo-code Transaction Handler"
+    ```
+    def handle_tx(request):
+      var body = request.getBody()
+      try:
+        var message = txMessageFromBody(body)
+        var tx = service.convertTransaction(txMessage)
+        var hash = tx.hash()
+        node.submitTransaction(tx)
+        return ok(createTxResponse(hash))
+      catch IllegalArgumentException as e:
+        return badRequest("Invalid request: " + e)
+      catch VerificationError as e:
+        return badRequest("Invalid transaction: " + e)
+      catch Error as e:
+        log("Internal error:" + e)
+        return internalServerError()
+      ```
+
+#### Executable Transactions
+
+To correctly process a transaction, it must be transformed into an
+*executable transaction* (see `TransactionConverter` or
+`Service#convertTransaction` method) and transmitted to the framework using
+`Node#submitTransaction` method. The framework checks it, and if it is correct,
+broadcasts it to other nodes of the system. Other nodes, having received the
+transaction message, convert it into an executable transaction, also using the
+service transaction converter.
+
+An executable transaction is a class object that implements
+`com.exonum.binding.messages.Transaction` interface and defines transaction
+behavior and business logic. The interface implementations must define the
+transaction authentication rule (usually, the digital signature verification of
+the message) - `isValid` method; and the execution rule for the
+transaction - `execute` method.
+
+Ed25519 is a standard cryptographic system for digital signature of Exonum
+messages. We use `com.exonum.binding.crypto.CryptoFunctions#ed25519`
+method for obtaining the implementation of the function.
+
+The implementation of `Transaction#isValid` transaction authentication method
+must be a pure function, i.e. for the given transaction to return the same
+result on all nodes of the system. For this reason, access to the Exonum
+storage by external files or network resources in method implementations is
+not allowed.
+
+`Transaction#execute` method describes the operations that are applied to the
+current storage state when the transaction is executed. Exonum transfers `Fork`
+as an argument - a view that allows performing modifying operations. A service
+schema object can be used to access data collections of this service.
+
+!!! note
+    `Transaction#info` will be deleted soon.
```

Comment:
Hm, `info` has been removed from the core for a while; is there a particular reason to keep it in the binding?

Reply:
It was renamed in the core, but in EJB we still use this name. By 'it will be deleted' I mean 'we consider getting rid of JSON altogether in the core framework'